spec/plotting/dataframe_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'spec_helper.rb'
+
+class Nyaplot::DataFrame
+  # Because it does not allow to any equality testing
+  def == other
+    other.is_a?(Nyaplot::DataFrame) && rows == other.rows
+  end
+end
+
+# FIXME: just guessed specs logic from code. As far as I can understand,
+# it can be broken in number of ways with incorrect arguments.
+#
+describe Daru::DataFrame, 'plotting' do
```

Comment:
Can we have dataframe plotting specs in a separate file? This file is getting too long anyway.